code/modules/atmospherics/gasmixtures/gas_types.dm

```diff
@@ -28,6 +28,7 @@ var/list/hardcoded_gases = list("o2","n2","co2","plasma") //the main four gases,
 	var/name = ""
 	var/gas_overlay = "" //icon_state in icons/effects/tile_effects.dmi
 	var/moles_visible = null
+	var/dangerous_concentration = 0 // levels at which Bad Shit(tm) starts to happen
```

Comment:
it is in preparation for a refactor of scanners that I am planning to do separately because this shit edits 4 maps